pkg/credhub/interpolate.go

```diff
@@ -0,0 +1,23 @@
+package credhub
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"code.cloudfoundry.org/buildpackapplifecycle/credhub"
```

Comment:
Do we really want a dependency between the lifecycles? For `dockerapplifecycle` there is no such dependency, right?

Reply:
Ya I was trying to reduce some code duplication, but I can just reimplement the function like the dockerapplifecycle does. Perhaps long term it makes sense to have a common library like libcnb for the lifecycles to use.

Reply:
I had a look at the `dockerapplifecycle` and I really like it. It doesn't introduce this dependency and I think it is easier to understand what is going on.